oxidation/libparsec/crates/core_fs/src/sync_transactions.rs

```diff
@@ -0,0 +1,121 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+use std::collections::HashSet;
+
+use libparsec_types::{BlockAccess, EntryID, FileManifest, FolderManifest, WorkspaceManifest};
+
+#[derive(Default)]
+pub struct ChangesAfterSync {
+    pub added_blocks: HashSet<BlockAccess>,
+    pub removed_blocks: HashSet<BlockAccess>,
+    pub added_entries: HashSet<EntryID>,
+    pub removed_entries: HashSet<EntryID>,
+}
+
+impl From<(&FileManifest, &FileManifest)> for ChangesAfterSync {
+    fn from(value: (&FileManifest, &FileManifest)) -> Self {
+        let old_manifest = value.0;
+        let new_manifest = value.1;
```

Comment:
```suggestion
    fn from((old_manifest, new_manifest): (&FileManifest, &FileManifest)) -> Self {
```

WDYT of directly deconstructing the tuple in the param ?

Reply:
:+1:

Reply:
Done